docs/design/StratisSoftwareDesign.lyx

```diff
@@ -2302,7 +2302,7 @@ description
 \begin_inset Text
 
 \begin_layout Plain Layout
-IEEE CRC32 of signature block (bytes at offset 4 length 508)
+IEEE CRC32C of signature block (bytes at offset 4 length 508)
```

Comment:
We will need to change implementation to reflect this change.

Reply:
I assumed we already were using crc32c? But if not, then yes. Also, there's a new field in static header we will need to read/write.

Reply:
Nope, we are using ieee.

Reply:
I will do it now, one fewer issue to write up.